Update length.adoc (#859)
arduino/reference-en: Language/Variables/Data Types/String/Functions/length.adoc

[float]
=== Returns

> The length of the String in characters as an unsigned int.

Reviewer: ```suggestion
The length of the String in characters. Data type: `unsigned int`.
```

This is the established convention for documenting return types.

Example: https://github.com/arduino/reference-en/blob/master/Language/Functions/Advanced%20IO/pulseIn.adoc#returns
Which extrapolates on the official reference sample, which doesn't provide guidance re: return type (https://github.com/arduino/reference-en/issues/199), but does for parameter type:
https://github.com/arduino/reference-en/blob/master/AsciiDoc_sample/Reference_Terms/AsciiDoc_Template-Single_Entity.adoc#parameters
